<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QWidget" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="SudokuCell" name="cell11">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell12">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell13">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell22">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell23">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell31">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell33">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell32">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell42">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell51">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell53">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell52">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell61">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell63">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell41">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell62">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell43">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell83">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell72">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell81">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell82">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell91">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell92">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell71">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell93">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell73">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell15">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell85">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell96">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell24">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell26">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell25">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell84">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell56">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell34">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell36">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell75">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell45">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell14">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell54">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell55">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell86">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell64">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell94">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell65">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell35">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell16">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell95">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell44">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell66">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell46">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell76">
   <property name="geometry">
    <rect>
     <x>217</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell74">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell18">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell88">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell99">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell27">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell29">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell28">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>61</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell87">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell59">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell37">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell39">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell78">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell48">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell17">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell57">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell58">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>186</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell89">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>311</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell67">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell97">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell68">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell38">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>102</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell19">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>20</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell98">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>352</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell47">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell69">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>227</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell49">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>145</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell79">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="SudokuCell" name="cell77">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>270</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonUndo">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>UNDO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPause">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>80</y>
     <width>71</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>PAUSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHint">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>80</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>HINT</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="toggleNotesMode">
   <property name="geometry">
    <rect>
     <x>403</x>
     <y>140</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <property name="text">
    <string>NOTES MODE</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>230</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton3">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton6">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>290</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton5">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton4">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>290</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton9">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>350</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLabel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mistakeCntLabel">
   <property name="geometry">
    <rect>
     <x>399</x>
     <y>170</y>
     <width>180</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SudokuCell</class>
   <extends>QLineEdit</extends>
   <header location="global">sudokucell.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
